help/en/releasenotes/current-draft-note.shtml

```diff
@@ -639,7 +639,11 @@
    <h3>Web Server</h3>
         <a id="server" name="server"></a>
         <ul>
-            <li></li>
+            <li>Added a new "web/idtags_examples" directory with an example
+                of  as icons, which move to different places on the webpage depending on their most recent report. 
```

Comment:
I have difficult to understand this sentence.

I'm not a native English speaker, but I wonder if maybe "of as" should be replaced with "with"?

... an example with icons, which move ...